src/Microsoft.IdentityModel.Tokens/TokenValidationParameters.cs

```diff
@@ -96,6 +96,12 @@ namespace Microsoft.IdentityModel.Tokens
     public delegate SecurityToken SignatureValidator(string token, TokenValidationParameters validationParameters);
 
     /// <summary>
+    /// Definition for TokenReader.
+    /// </summary>
+    /// <param name="token">A securityToken with a signature.</param>
+    public delegate SecurityToken TokenReader(string token);
+
+    /// <summary>
```

Comment:
In the ctor: TokenValidationParameters(TokenValidationParameters other), we need to set TokenReader to other.TokenReader

Reply:
Sorry for missing this. Just fixed it.